client/my-sites/upgrades/checkout/credits-payment-box.jsx

```diff
@@ -46,14 +46,9 @@ var CreditsPaymentBox = React.createClass( {
 	},
 
 	render: function() {
-		var classSet = classNames( {
-			'credits-payment-box': true,
-			selected: this.props.selected === true
-		} );
-
 		return (
 			<PaymentBox
-				classSet={ classSet }
+				classSet="credits-payment-box"
```

Comment:
It becomes tough separate this code because is part of the goal to achieve: "do not use css class to hide components" and it's exactly this class that hides the components.